src/gsi-ncdiag/combine_files.py

```diff
@@ -12,10 +12,10 @@
 import datetime as dt
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
-if not IODA_CONV_PATH.is_dir():
-    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
-sys.path.append(str(IODA_CONV_PATH.resolve()))
+IODA_FILES_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
```

Comment:
This CONV is for converters not, conventional, need to change these 4 lines back.

Reply:
Oh, I see now.